### DISCUSSION TITLE
Additional Mob Spawner Params

### DISCUSSION THREAD

**Discussion by James-P-Bennett** [PurpurMC/Purpur#1585] [Feature Requests]
I'm not sure if this would be practical for improving server performance, but I think it would be useful to have a configurable option to increase the range at which spawners spawn more mobs during their spawn tick.

Currently, spawners check for 4 mobs within a 4-block horizontal and 1-block vertical range by default. A spawner activates when a player (not in Spectator mode) is within a 16-block spherical radius from the center of the spawner.

Allowing adjustments to these parameters could provide additional control over mob spawner farms and dungeon spawning on servers.

**Comment by James-P-Bennett** [comment]
Other decent ideas:
https://github.com/PurpurMC/Purpur/discussions/1110
and
https://github.com/PurpurMC/Purpur/discussions/927

**Comment by YouHaveTrouble** [comment]
The parameters you listed can already be modified with plugins or/and commands

**Reply by James-P-Bennett** [reply]
Alright, but for instance, wouldn’t modifying a vanilla spawner tick with a plugin be less efficient compared to letting the server handle it directly? Or is the difference negligible?

(By the way, As mentioned I’m looking for a plugin that can adjust the range at which mob spawners detect already spawned mobs. If you know of one, let me know.) 😁

**Reply by YouHaveTrouble** [reply]
It likely wouldn't make a difference in the long run.
